infrastructure/main.tf

```diff
@@ -3,7 +3,7 @@ provider "azurerm" {
 }
 
 locals {
-  aseName = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  aseName = "core-compute-${var.env}"
```

Comment:
can you remove the remote state reference in state.tf too?

Reply:
forgot about this!